collect_app/src/main/java/org/odk/collect/android/receivers/NetworkReceiver.java

```diff
@@ -61,20 +61,16 @@ public void onReceive(Context context, Intent intent) {
 
         NetworkInfo currentNetworkInfo = manager.getActiveNetworkInfo();
 
-        if (action.equals(ConnectivityManager.CONNECTIVITY_ACTION)) {
+        if (action.equals(ConnectivityManager.CONNECTIVITY_ACTION)
```

Comment:
Phew 😄  I read this method a bunch of times trying to figure out why it was acquiring the ConnectivityManager twice.  Reassuring to know that it was just unnecessary duplication.